runtime/jslib.js

```diff
@@ -24,7 +24,7 @@ function caml_js_pure_expr (f) { return f(); }
 
 //Provides: caml_js_set (mutable, const, const)
 function caml_js_set(o,f,v) { o[f]=v;return 0}
-//Provides: caml_js_get mutable (const, const)
+//Provides: caml_js_get mutator (const, const)
```

Comment:
Shouldn't we declare caml_js_delete (just below) also mutator?

Reply:
The absence of an annotation is actually interpreted as "mutator":
https://github.com/ocsigen/js_of_ocaml/blob/8c993985feb70314635d83d31ea7c0daa10efe37/compiler/lib/annot_parser.mly#L36 so I actually removed the annotation from `caml_js_get` for uniformity with the rest of the file.